pkg/controller/servicebinding/retriever.go

```diff
@@ -125,10 +124,10 @@ func (r *retriever) ProcessServiceContexts(
 		if err != nil {
 			return nil, nil, err
 		}
+		volumeKeys = append(volumeKeys, v...)
```

Comment:
is this change no-op? Having it at line 127 has the same effect as leaving it at line 131?

Reply:
yes

Reply:
Done. 172198aa20026266b5b017693826e754be0f1567